Styling/README.md

```diff
@@ -501,6 +502,25 @@ Returns the cosine value of the number (in radians).
 }
 ```
 
+#### sqrt 
+
+`sqrt(value : Number) : Number`
+
+Returns the square root of value if value >= 0. Returns NaN when value < 0.
+
+```json
+{
+    "color" : {
+    "expression" : "sqrt(${temperature}",
+    "conditions" : [
+        ["${expression} >= 0.5", "color('#00FFFF')"],
+        ["${expression} >= 0.0", "color('#FF00FF')"],
+        ["true", "color('#FFFFFF')"]
```

Comment:
You can remove this last line since it should always meet the two previous conditions.